eurostag-converter/src/main/java/com/powsybl/eurostag/converter/EurostagEchExport.java

```diff
@@ -562,9 +566,315 @@ private void createTransformers(EsgNetwork esgNetwork, EsgGeneralParameters para
             esgNetwork.addDetailedTwoWindingTransformer(esgTransfo);
         }
 
-        if (network.getThreeWindingsTransformerCount() > 0) {
-            //TODO
-            throw new UnsupportedOperationException("Three windings transformers are not supported yet");
+        for (ThreeWindingsTransformer t3wt : Identifiables.sort(network.getThreeWindingsTransformers())) {
+            if (config.isExportMainCCOnly() && !EchUtil.isInMainCc(t3wt, config.isNoSwitch())) {
```

Comment:
Do you really need to make this test twice?

Reply:
twice?